Assignment 3.Rmd

```diff
@@ -136,18 +144,84 @@ What patterns do you see in the plot?
 It would be useful to determine how many people are in each cluster. We can do this easily with dplyr.
 
 ```{r}
-K7 <- 
+K7 <- count(K4, cluster)
 ```
 
 Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. Which cluster grouping do you think is more informative? Write your answer below:
 
+I think 3 cluster is more informative because for although for both 2 and 3 clusters we can see the min and max range of the average motivation, 3 clusters gives us more information of inbetween the range.
```

Comment:
I'm not sure I understand what you mean by "more information" here? Can you explain a little more about what you mean by more informative?